src/components/Navigation.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="nav">
+      <div class="top-banner" :class="{ 'show': $route.path!=='/' }">
```

Comment:
Try to make the indentation more consistent (use two spaces).

Also for `$route.path!=='/'`, add spaces, make them into `$route.path !== '/'`